devops/ansible/roles/girder/library/test/run.sh

```diff
@@ -1,5 +1,5 @@
 #/bin/bash
 
 ansible-playbook -i .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory \
-                 --private-key=.vagrant/machines/girder/virtualbox/private_key \
-                 -u vagrant $*
```

Comment:
I went with tabs only because lots of `.bat` / `.sh` already had them. Upon your request, I've switched the rules to require 4 spaces for those too, but kept tabs for `Makefile`, just because it's technically required.